lib/backend/etcdbk/etcd.go

```diff
@@ -649,6 +649,60 @@ func (b *EtcdBackend) NewWatcher(ctx context.Context, watch backend.Watch) (back
 	return b.buf.NewWatcher(ctx, watch)
 }
 
+func (b *EtcdBackend) Iterate(ctx context.Context, startKey, endKey backend.Key, limit int, order backend.IterationOrder) (iter.Seq2[backend.Item, error], error) {
```

Comment:
How do you feel about having the `Iterate` method take a struct with named parameters instead?  Ex:

```golang
items := bk.Iterate(ctx, backend.Iteration{
    StartKey: myStartKey,
    EndKey: myEndKey,
})
```

Benefits include improved readability, easier future extension, and 99% of teleport usecases being able to omit order since order since we almost always want to default to ascending.